Plots/Masking/NCL_mask_5.py

```diff
@@ -0,0 +1,154 @@
+"""
+NCL_mask_5.py
+================
+
+This script illustrates the following concepts:
+    - Using cartopy.feature options to display map information
+    - Paneling two plots on a page with 'subplots' command
+    - Drawing contours
+    - Explicitly setting the fill colors for contours
+    - Drawing contours over land only
+    - Using draw order resources to mask areas in a plot
+    - Adding a label bar
+    - Implementing best practices when choosing a color scheme 
+
+    
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/mask_5.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/mask_5_lg.png
+                         
+"""
+###############################################################################
+# Import packages:
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import cartopy.feature as cfeature
+import matplotlib.patches as mpatches
+
+from geocat.viz import util as gvutil
+import geocat.datafiles as gdf
+###############################################################################
+# Plot
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset(gdf.get("netcdf_files/atmos.nc"), decode_times=False)
+t = ds.TS.isel(time=0)
+
+# Fix the artifact of not-shown-data around 0 and 360-degree longitudes
+wrap_t = gvutil.xr_add_cyclic_longitudes(t, "lon")
+                    
+###############################################################################
+#Plot:
+
+fig = plt.figure(figsize=(12, 14))
```

Comment:
This is where you should call `plt.subplots`. Notice that this function has an 's' at the end. It does something different than `plt.subplot` which is what you are using right now elsewhere in this file. You might want to use something similar to what I have below. You will likely need to change `figsize` and `hspace`, but this is the easiest way to stack two plots on top of each other. [Here](https://matplotlib.org/3.2.2/api/_as_gen/matplotlib.pyplot.subplots.html) is the documentation page for this function. The `gridspec_kw` gives you a lot of control over the layout.
```fig, ax = plt.subplots(2, 1, figsize=(6, 10), gridspec_kw=dict(hspace=0.3), subplot_kw={"projection": projection})```

Reply:
So I've been playing around with using subplots over subplot and the biggest issue I've been coming across is that it seems when using subplots, you need to reference data that is being displayed on the project, but I'm not sure if this is 100% the case (there aren't many online sources talking about the differences between the two and when to use one over the other).  The issue I am seeing comes with having one plot that uses no data (masking only) and the other is using data (masking with data). I did try using a couple of the panel plots we already have and some of the examples from the documentation page as a reference to what to do, but even within those they are using same source but plotting different data. I may keep playing with it for a bit longer, but if I can't resolve this issue, I will most likely just use the original work around I have for this plot.

Reply:
Using what you have is perfectly fine just make sure you add comments thoroughly explaining why the `row` argument is what it is in lines 99 and 153 since it seems to be a "hacky" solution.

I'm not sure if I fully understand your issue with needing to "to reference data that is being displayed on the project". If you'd like to meet and talk about to try and find a simpler solution, I'm open to that. I've made quite a few panel plots in this project and in others, so maybe I can help.